TROUBLESHOOTING.md

```diff
@@ -0,0 +1,155 @@
+# Troubleshooting performance issues
+
+Performance is at the heart of Quarkus.
+
+If you are facing performance issues (runtime or startup issues), and would like to discuss them with the Quarkus Team,
+you are more than welcome on our [mailing list](https://groups.google.com/d/forum/quarkus-dev),
+[Zulip chat](https://quarkusio.zulipchat.com) or [Github issue tracker](https://github.com/quarkusio/quarkus/issues).
+
+To help us to troubleshoot your issues, we will need some performance insight from your application.
```

Comment:
```suggestion
To help us troubleshooting your issues, we will need some performance insights from your application.
```

Reply:
Gracie plugin of IntelliJ suggests to replace troubleshooting by to troubleshoot. But maybe it's a difference between english english and american english ?

Reply:
Maybe @geoand can share some light on `to troubleshoot` vs `troubleshooting` ?

Reply:
I am definitely not an English grammar expert (having grown up bilingual but crucially, not having attended English speaking school), but for the most corrent way of saying it would be:

```
To help us troubleshoot
```

Reply:
OK, done.